From 9480624cae7e5d70c5482ed37783c727f3bcd989 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 10 Jun 2024 10:13:31 +0200
Subject: [PATCH 0925/1695] dts: stm32: remove include of RIF* bindings in
 stm32mp21x dt files

Since it is included at SoC level, remove all includes of these bindings
at board level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 60bd9abff8e95c3cb54daac943f45a577ae95393 dts: stm32: remove include of RIF* bindings in stm32mp21x dt files]
Change-Id: I3ce457a9ac3cd1c07f16b4647bbe3650817a39bb
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/386449
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index e2d90e888..97d496b55 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -6,10 +6,6 @@
 /dts-v1/;
 
 #include <dt-bindings/clock/stm32mp21-clksrc.h>
-#include <dt-bindings/soc/stm32mp21-rifsc.h>
-#include <dt-bindings/soc/stm32mp25-rif.h>
-#include <dt-bindings/soc/stm32mp25-risab.h>
-#include <dt-bindings/soc/stm32mp25-risaf.h>
 #include "stm32mp215.dtsi"
 #include "stm32mp21xf.dtsi"
 #include "stm32mp215f-dk-ca35tdcid-rcc.dtsi"
-- 
2.47.2

